pyproject.toml

```diff
@@ -58,6 +58,9 @@ Documentation = "https://docs.openmc.org"
 Repository = "https://github.com/openmc-dev/openmc"
 Issues = "https://github.com/openmc-dev/openmc/issues"
 
+[tool.setuptools.dynamic]
+version = {file = "tools/version.txt"}
```

Comment:
Does this file need locating elsewhere, perhaps in the source code. I'm not sure but perhaps that helps when packaging as I don't think the tools folder is included in the packaging (e.g. conda)

Reply:
This file is only required at build time. If it is necessary at runtime, we can put it in the root dir.

Reply:
It appears that **openmc-feedstock** uses a specific [release tar file](https://github.com/conda-forge/openmc-feedstock/blob/main/recipe/meta.yaml#L23), which includes all the folders, including the tools folder.

Reply:
Thanks for answering this

Reply:
I have stored this file in the tools directory because it is only required during build time. Python's wheel uses it to create the wheel, and CMake uses it to generate the `version.h` file. Additionally, we can use this file for other purposes. I placed it in that folder to keep the root directory clean. However, if necessary, we can move it to the root directory.

Reply:
I tend to agree that this is more of a root-level file.